catalog/plugin/catalog-plugin-metacard-validation/src/test/java/ddf/catalog/metacard/validation/MetacardValidityCheckerPluginTest.java

```diff
@@ -529,6 +545,20 @@ public void testPropertyIsLike() {
     }
 
     @Test
+    public void testPropertyIsLikeTrue() {
+        assertThat(
+                testValidationQueryDelegate.propertyIsLike(VALIDATION_ERRORS, "sample-validator", true),
+                is(true));
+    }
+
+    @Test
```

Comment:
This test name is a little weird. Maybe you could rename it to something like `testPropertyIsLikeTrueWithWarnings` or something like that.